.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+pnpm lint-staged
```

Comment:
Should we add prettier and typecheck for pre-commit hooks ?

Reply:
Done